{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -48,22 +55,19 @@ maintainers = [
 
 [project.optional-dependencies]
 doc = [
-    'sphinx>=4.1.1',
-    'sphinx-argparse',
-    'sphinx-autodoc-typehints',
-    'sphinxcontrib-httpdomain',
-    'sphinxcontrib-openapi',
-    'sphinxcontrib-serializinghtml',
-    'sphinxcontrib-websupport',
-    'sphinx_rtd_theme',
+    "sphinx>=4.1.1",
+    "sphinx-argparse",
+    "sphinx-autodoc-typehints",
+    "sphinx_rtd_theme",
+    "sphinxcontrib-openapi",
+    "sphinxcontrib-websupport",
     "numpydoc>=1.7"
 ]
 test = [
     "pytest>=3",
     "pytest-cov",
     "pytest-reporter-html1",
-    "urlchecker",
-    "gevent"
```

Comment:
You do not need gevent? It is required by the coverage, right?
```
.coveragerc 
```

Reply:
Good point! I forgot that.